scripts/generate-authors.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -e
+
+cd "$(dirname "$(readlink -f "$BASH_SOURCE")")/.."
+
+# see also ".mailmap" for how email addresses and names are deduplicated
+{
+	cat <<-'EOF'
+	# This file lists all individuals having contributed content to the repository.
+	# For how it is generated, see `scripts/generate-authors.sh`.
+	EOF
+
+	# old manual entries 
+	read -d '' authors <<-"EOF" || true
+	Michel Baylac <>
+	Cyrille d'Haese <>
+	Ellen Reitmayr <>
+	Michael Beckmann <>
+	Oliver Beckmann <>
+	Fedor Bezrukov <>
+	Fabian Bieker <>
+	François Charette <>
+	Aaron Chen <>
+	Fabrice Dessaint <>
+	Nathan Dunn <>
+	Alexis Gallagher <>
+	Stefano Gariazzo <>
+	David Gleich <>
+	Bernd Kalbfuss <>
+	Martin Kähmer <>
+	Ervin Kolenovic <>
+	Krzysztof A. Kościuszkiewicz <>
+	Christian Kopf <>
+	Jeffrey Kuhn <>
+	Uwe Kuehn <>
+	Felix Langner <>
+	Stephan Lau <>
+	Alex Montgomery <>
+	Saverio Mori <>
+	Ambrogio Oliva <>
+	Stephan Rave <>
+	John Relph <>
+	Hannes Restel <>
+	Moritz Ringler <>
+	Rudolf Seemann <>
+	Toralf Senger <>
+	Manuel Siebeneicher <>
+	Mike Smoot <>
+	Ulrich Stärk <>
+	Martin Stolle <>
+	David Weitzman <>
+	John Zedlewski <>
+	Samin Muhammad Ridwanul Karim <>
+	Stefan Robert <>
+	EOF
+
+	echo -e "$authors\n$(git log --format='%aN <%aE>')" | LC_ALL=C.UTF-8 sort -uf
```

Comment:
Can you please use the full options for sort -uf, and place a comment what LC_ALL=C.UTF-8 is for. 
Maybe also explain the format string used in the log statement.